gulpfile.babel.js

```diff
@@ -50,12 +76,15 @@ const buildConfig = (invocationArgs, publicRoot, sourceRoot, testRoot, exportRoo
   config.dir.src.fonts = `${config.sourceRoot}/fonts`;
   config.dir.src.templates = `${config.sourceRoot}/_patterns`;
   config.dir.src.js = `${config.sourceRoot}/js`;
+  config.dir.src.jsMap = `${config.sourceRoot}/js`;
 
   config.dir.test.sass = `${config.testRoot}/sass`;
+  config.dir.test.js = `${config.testRoot}/app`;
```

Comment:
```suggestion
  config.dir.test.js = `${config.testRoot}/js`;
```

?

Reply:
Favoured `/app` over `/js` to try to avoid ambiguity, as sibling directory `/libero-config` also contains js tests.

Reply:
As that will disappear at some point, I wouldn't worry about it.

Reply:
Fair.